packages/nimble-components/src/select/styles.ts

```diff
@@ -28,6 +40,81 @@ export const styles = css`
     [part='end'] {
         display: contents;
     }
+
+    .listbox {
+        overflow-x: clip;
+    }
+
+    .listbox.empty slot {
+        display: none;
+    }
+
+    .listbox.above {
+        flex-flow: column-reverse;
+    }
+
+    .search-field {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        height: ${controlHeight};
+        background: transparent;
+    }
+
+    .search-icon {
+        padding-left: ${smallPadding};
+    }
+
+    .search-field::after,
+    .search-field::before {
+        content: '';
+        position: absolute;
+        height: 0px;
+        border-bottom: rgba(${borderRgbPartialColor}, 0.15) 2px solid;
+    }
+
+    .search-field::after {
+        top: calc(${controlHeight} + ${smallPadding} - ${borderWidth});
+    }
+
+    .search-field:before {
+        bottom: calc(${controlHeight} + ${smallPadding} + (2 * ${borderWidth}));
+    }
+
+    .search-field:not(.above)::after,
+    .search-field.above::before {
+        width: calc(100% - (2 * ${borderWidth}));
+    }
+
+    .filter-input {
+        background: transparent;
+        border: none;
+        color: inherit;
+        font: inherit;
+        height: var(--ni-nimble-control-height);
+        padding: 0 ${smallPadding} 0 ${mediumPadding};
+        width: 100%;
+    }
+
+    filter-input::placeholder {
```

Comment:
I can't tell that this styling is working as expected. Shouldn't it be the `filter-input` class that is styled?
```suggestion
    .filter-input::placeholder {
```